openmc/data/photon.py

```diff
@@ -379,6 +391,41 @@ def to_hdf5(self, group, shell):
                  _SUBSHELLS, range(len(_SUBSHELLS)))
             group.create_dataset('transitions', data=df.values.astype(float))
 
+    def energy_fluorescence(self, shell):
+        """Compute expected energy of fluorescent photons for the shell
+        """
+
+        if shell not in self.binding_energy:
+            raise KeyError('Invalid shell {}.'.format(shell))
+
+        if shell in self._e_fluorescence:
+            # Already computed
+            return self._e_fluorescence[shell]
+        else:
```

Comment:
"Flat is better than nested" -- remove the `else:` and de-dent the block following.

Reply:
done